command_line/spot_counts_per_image.py

```diff
@@ -67,6 +67,12 @@ def run(args=None):
         sys.exit("Only one reflection list may be passed")
     reflections = reflections[0]
 
+    if "miller_index" in reflections:
```

Comment:
```
if "miller_index" in reflctions or any(experiments.crystals())
```
->  catch both outcomes (because I can see someone passing indexed.refl imported.expt or something in response to one of the cases)

if both then maybe 

```
sys.exit("Only unindexed experiments / reflections are currently supported")
```

So you catch this in one go?